include/zephyr/drivers/espi_emul.h

```diff
@@ -1,185 +0,0 @@
-/*
```

Comment:
Please don't delete the file, we should instead leave an empty stub here with a warning that has:

```
#warning "<zephyr/espi_emul.h> is deprecated, please include <zephyr/espi/espi_emul.h>"
#include <zephyr/espi/espi_emul.h>
```